project-proposal.md

```diff
@@ -18,51 +18,63 @@ The main goal of our project is to make it easier on both those getting into rea
 
 The system will be tested in parts. First step will be to ensure that we can properly retrieve book data from the Google Books API via the REST API that we will implement. Second we will test that we can serve that data up to the frontend application on demand. Third, we will make sure we can display that info to the user from the frontend application.
 
+
 ### Tools:
 
-* Frontend: JS Framework
-* API: Java or Python
+* Frontend: Ember
+* API: Java
 * Database: N/A, API allows 1000 reqs/day
 * IDE: VS Code - Frontend | IntelliJ - Backend
 
 ## Team
 
 ### Skills:
 
-Logan and Camille have experience using Ember.js to build frontends, as well as Java to build APIs. Logan also has experience with Oracle databases and Microsoft SQL Server. Braden Martin has basic web development experience (HTML, CSS, JS), as well as experience working with SQL databases. Grant has experience with Python for frontend applications and Java APIs. (Other people insert your experiences). None of us have used (JS framework if not ember) before, but we should be able to adapt given our individual experiences. We’re familiar with (backend language) but have never built an API from scratch.
+Logan and Camille have experience using Ember.js to build frontends, as well as Java to build APIs. Logan also has experience with Oracle databases and Microsoft SQL Server. Braden Martin has basic web development experience (HTML, CSS, JS), as well as experience working with SQL databases. Grant has experience with Python for frontend applications and Java APIs. Braden Butler has experience with C and C++. We’re familiar with Java APIs but have never built one from scratch.
 
 ### Roles:
 
 * Grant - (Frontend)
-* Logan - (Frontend)
-* Camille - (Backend)
-* Braden M. - (Backend (API requests))
-* Braden B. - (Backend)
+* Logan - (FBackend, Team Leader)
+* Camille - (Backend % Frontend, Customer)
+* Braden M. - (Backend, Customer)
+* Braden B. - (Frontend)
 
 ## Project Management
 
 ### Schedule:
 
 Completion of this project is feasible. We plan on meeting once a week over (Zoom/Discord) to give an update on the progress we’ve made. We will have constant chats over Discord for smaller questions/suggestions. We will develop a full schedule and timeline with milestones in our final project proposal.
 
+
+2/19-2/25          Work Begins: Select programming languages, learn basics, map out feature list
+2/26-3/4           Second Sprint: Begin work on GUI, Randomize Google Books API
+3/5-3/11           Iteration 1 Status Report
+3/12-3/18          Third Sprint: Add design to GUI, Add filtering to Google Books API
+3/19-3/25          Iteration 2 Status Report
+3/26-4/1           Fourth Sprint: Implement functionality into GUI, clean up code
+4/2-4/8            Iteration 3 Status Report
+4/9-4/14           Final Sprint: Bug fixes, design tweaks, bonus features
+4/15               Submission and Presentation
```

Comment:
I think using a markdown table to display this chart would make it more clear. Other than that, it looks great!